src/Chapter06/Listing06.26.DefiningAPrimaryConstructor.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_26;
+
+#if NET8_0_OR_GREATER
+public class Program
+{
+    public static void Main()
+    {
+        Employee employee;
+        employee = new("Inigo", "Montoya");
```

Comment:
Any reason to not combine declaration and assignment?
```suggestion
        Employee employee = new("Inigo", "Montoya");
```